cmd/litcli/autopilot.go

```diff
@@ -65,6 +66,12 @@ var addAutopilotSessionCmd = cli.Command{
 				"perform actions on. In the " +
 				"form of: peerID1,peerID2,...",
 		},
+		cli.StringSliceFlag{
+			Name: "configuration",
+			Usage: `per feature JSON-serialized configuration, ` +
+				`expected format: {"version":0,"option1":` +
+				`"parameter1","option2":"parameter2",...}`,
```

Comment:
the `configuration` flag will apply to all features though right? so should the config values not be indexed by feature name? else we send the same config to all features we subscribe to